### PR TITLE
Added navigation component

### PR DESCRIPTION
**Pull request by Shah-Arsalan**
**Folder Structure:**
Component Folder 
Drawer Folder 
Images Folder 
document.css
document.html
index.css
index.html
main.css

Go to the the component folder to get the CSS of all the individual components. 
HTML of all the components is to be found in document.html ( scroll down to get the relevant component)

### CODE REVIEW

**Review by thegeorgenikhil — APPROVED** [review]
Beautify/prettify the code before pushing, too many line breaks in the html file

File: components/header.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+.nav-container-short-always-collapsed{
+    display: flex;
+    background-color: var(--primary-color);
+width: 7rem;
+padding: 1rem;
+justify-content: space-evenly;
+border-bottom-right-radius: 1rem;
+color: var(--white)
+}
+
```
Comment (anuj-kumary) [inline components/header.css:50]: Format document so it look good

File: document.html [hunk 2]
```diff
@@ -534,6 +534,97 @@ <h1 class="large-heading-text margin-1rem">Inputs</h1>
...
+
+                        <div>
+
+                            <i class="fas fa-bars pointer"></i>
+
+                            <h2 class="title">San Francisco, CA</h2>
+
+
+                        </div>
+
```
Comment (anuj-kumary) [inline document.html]: Remove this all extra spaces

**Review by vinc3nati — APPROVED** [review]
Use `prettier` extension for formatting, else good to proceed!

**Review by lomesshh — APPROVED** [review]

File: components/header.css (hunk 1 above)
Comment (Shah-Arsalan) [reply components/header.css:50]: done

File: document.html (hunk 2 above)
Comment (Shah-Arsalan) [reply document.html]: done

**Pull request merged**